### ISSUE TITLE
[MPOM-1] apache-release does unwanted things in the prepare goal

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-apache-parent#309]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** opened **[MPOM-1](https://issues.apache.org/jira/browse/MPOM-1?redirect=false)** and commented

Release 7 of the org.apache:apache use -P to turn on an 'apache-release' profile in the release plugin. By using -P, this profile is turned on in prepare as well as perform.

This may turn into a contest of opinion, and you all may just tell me to jump in a lake, but I think this is a bad idea. At prepare time, we're just looking to get the versions setup and the tag made. Having to deal with gpg signing, and source archiving seems out of place.

At very least, I'd request that this the \<arguments/> element be left out of the release plugin spec. If some project wants to use that particular collection of functions, fine. For the rest of us, who just want to get distribution management and other universals, you we wouldn't have to worry about it.

---

**Affects:** ASF-7

**Comment by jira-importer** [comment]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** commented

fixed rev 1065772

**Comment by jira-importer** [comment]
**[Brian E Fox](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brianf)** commented

This needs a different approach. If people don't want to use the profile by default, then lets put the profile argument into a property that can easily be overridden. It is very important though that we have proper release proceedures embodied by default in the ASF pom. People shouldn't have to customize sections of their poms to get proper releases. As it was done in the previous release, it "just works"

**Comment by jira-importer** [comment]
**[Brian E Fox](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brianf)** commented

A little history on this setup:
It was moved to the ASF pom from the Maven pom when we tried to provide correct defaults for all ASF projects built with Maven after a lengthy discussion on legal about minimal release requirements. The conclusion was to encapsulate best practices so projects that aren't maven experts can still get meet the release requirements.

This particular configuration of the release plugin dates back to the very first setup of the release profiles in the Maven pom, specifically added here: http://svn.apache.org/viewvc?view=revision&revision=502422 a little over 4 years ago.
